src/components/Footer.jsx

```diff
@@ -111,4 +109,4 @@ const Footer = () => {
   );
 };
 
-export default Footer;
+// export default Footer;
```

Comment:
- [ ] Commenting the Footer component is adding lots of errors to the codebase making it difficult to review your code. Kindly remove all commented code that leads to errors or bugs

![image](https://user-images.githubusercontent.com/63473584/203520979-4bc84010-b728-40f9-a096-d88140f88ac6.png)